app/models/user.rb

```diff
@@ -14,6 +14,7 @@ class User < ActiveRecord::Base
   validates :name, presence: true, length: { in: 2..32 }
   validates :username, presence: true
   validates :email, presence: true
+  validates_format_of :email, :with => /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i
```

Comment:
Line is too long. [84/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, format: value` over `validates_format_of`.